README.md

```diff
@@ -1,5 +1,5 @@
-[![Build Status](https://travis-ci.org/mozfreddyb/eslint-plugin-no-unsafe-innerhtml.svg?branch=master)](https://travis-ci.org/mozfreddyb/eslint-plugin-no-unsafe-innerhtml)
-# Disallow unsafe HTML templating (no-unsafe-innerhtml)
+[![Build Status](https://travis-ci.org/mozilla/eslint-plugin-no-unsanitized.svg?branch=master)](https://travis-ci.org/mozilla/eslint-plugin-no-unsanitized)
+# Disallow unsanitized DOM access (no-unsanitized)
```

Comment:
We should move from saying it is DOM access as a primary header (happy for this to be a follow up)